ansible/group_vars/proxy-hosts.yml

```diff
@@ -12,16 +12,25 @@ nginx_proxy_worker_processes: 4
 nginx_proxy_buffers: 32 16k
 
 nginx_proxy_ssl: True
-nginx_proxy_ssl_certificate: /etc/nginx/ssl/idr-fullchain.pem
-nginx_proxy_ssl_certificate_key: /etc/nginx/ssl/idr-privkey.pem
+nginx_proxy_ssl_certificate: "{{ idr_default_nginx_proxy_ssl_certificate | default('/etc/nginx/ssl/idr-fullchain.pem') }}"
+nginx_proxy_ssl_certificate_key: "{{ idr_default_nginx_proxy_ssl_certificate_key | default('/etc/nginx/ssl/idr-privkey.pem') }}"
 nginx_proxy_http2: True
 
 
 ######################################################################
 # OMERO.web proxy
 
+nginx_proxy_log_format: main_timed_cache_upstream
+
 nginx_proxy_websockets_enable: True
 
+nginx_proxy_upstream_servers:
+- name: omeroreadonly
+  balance: ip_hash
```

Comment:
`ip_hash` is the only way in the open-source version of Nginx to ensure a client with a session is sent to the same backend, but I think Nginx should detect a backend is down and automatically "recalculate" the hash.

Haproxy and the commercial version of Nginx Plus both have proper support for HTTP session pinning.